src/lib/agent/PolykeyAgent.ts

```diff
@@ -63,6 +90,7 @@ class PolykeyAgent {
   // For storing the state of each polykey node
   // Keys are the paths to the polykey node, e.g. '~/.polykey'
   private polykeyMap: Map<string, Polykey> = new Map();
```

Comment:
I believe we should be doing 1 agent - 1 store.

Separate agents for separate stores.

Reply:
https://github.com/MatrixAI/js-polykey/issues/105